### DISCUSSION TITLE
Videos gallery

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#9391] [Q&A / Support]
Just as it is possible to create a photo gallery for a given object, is it possible to create a video gallery for the same object?
Because I have the impression that I can put only one video.

**Comment by MadeP** [comment]
As far as i know, there is no video gallery. You could use object bricks for this. Create an objectbrick with a video data component and add it to your class. You can then also specify the maximum number of videos.

**Reply by weisswurstkanone** [reply]
Maybe use a block as a container
